### ISSUE TITLE
New sddm bundle issue

### ISSUE THREAD

**Issue by 0n0w1c** [clearlinux/distribution#1251]
Upon installing the newly created separate bundle for sddm, I have the following in the xorg log.

`[  1895.230] (EE) Failed to open authorization file "/var/run/sddm/{85a589ae-514b-4d03-9e4c-778259068d09}": No such file or directory`

`$ ls -l /var/run/sddm/{85a589ae-514b-4d03-9e4c-778259068d09}
-rw------- 1 sddm sddm 52 Sep 19 09:51 /var/run/sddm/{85a589ae-514b-4d03-9e4c-778259068d09}`

If I search, I find others that have run into this issue but no good solution that I have found yet.
[https://www.reddit.com/r/openSUSE/comments/b1a3fe/sddmhelper_problem/](url)
[https://forums.opensuse.org/showthread.php/535271-SDDM-authorization-fails](url)

I have openbox configured as the window manager. After entering a password and pressing [Enter], I can operate openbox but the SDDM background never clears. Opened window frames smear across the screen if moved. So the login does occur, just sddm is not getting out of the way.

If I use lightdm, logins work as expected. Also, I can successfully start X via startlxqt

**Comment by 0n0w1c** [comment]
I should add that I do not have KDE installed. My understanding is this sddm bundle was to be built without the Plasma display. I have installed the lxqt-desktop and using openbox as the WM.

**Comment by 0n0w1c** [comment]
$ cat /var/log/sddm.log
`[10:23:31.016] (II) DAEMON: Initializing...`
`[10:23:31.020] (II) DAEMON: Starting...`
`[10:23:31.021] (II) DAEMON: Logind interface found`
`[10:23:31.022] (II) DAEMON: Adding new display on vt 1 ...`
`[10:23:31.022] (II) DAEMON: Loading theme configuration from ""`
`[10:23:31.022] (II) DAEMON: Display server starting...`
`[10:23:31.022] (II) DAEMON: Running: /usr/bin/X -nolisten tcp -auth /var/run/sddm/{79f620a4-ada1-4881-837e-087ef36f94e3} -background none -noreset -displayfd 17 -seat seat0 vt1`
`[10:23:31.658] (II) DAEMON: Setting default cursor``
`[10:23:31.681] (II) DAEMON: Running display setup script  "/usr/share/sddm/scripts/Xsetup"`
`[10:23:31.684] (II) DAEMON: Display server started.`
`[10:23:31.684] (II) DAEMON: Socket server starting...`
`[10:23:31.684] (II) DAEMON: Socket server started.`
`[10:23:31.684] (WW) DAEMON: The configured theme "breeze" doesn't exist, using the embedded theme instead`
`[10:23:31.684] (II) DAEMON: Loading theme configuration from ""`
`[10:23:31.685] (II) DAEMON: Greeter starting...`
`[10:23:31.685] (II) DAEMON: Adding cookie to "/var/run/sddm/{79f620a4-ada1-4881-837e-087ef36f94e3}"`
`[10:23:31.700] (II) DAEMON: Greeter session started successfully`
`[10:23:31.763] (II) DAEMON: Message received from greeter: Connect`
`[10:23:40.396] (II) DAEMON: Message received from greeter: Login`
`[10:23:40.396] (II) DAEMON: Reading from "/usr/share/xsessions/lxqt.desktop"`
`[10:23:40.396] (II) DAEMON: Reading from "/usr/share/xsessions/lxqt.desktop"`
`[10:23:40.396] (II) DAEMON: Session "/usr/share/xsessions/lxqt.desktop" selected, command: "startlxqt"`
`[10:23:40.415] (II) DAEMON: Authenticated successfully`
`[10:23:40.433] (II) DAEMON: Auth: sddm-helper exited successfully`
`[10:23:40.433] (II) DAEMON: Greeter stopped.`
`[10:23:40.436] (II) DAEMON: Session started`
`[11:19:03.290] (WW) DAEMON: Signal received: SIGTERM`
`[11:19:03.296] (II) DAEMON: Socket server stopping...`
`[11:19:03.296] (II) DAEMON: Socket server stopped.`
`[11:19:03.296] (II) DAEMON: Display server stopping...`
`[11:19:03.508] (II) DAEMON: Display server stopped.`
`[11:19:03.508] (II) DAEMON: Running display stop script  "/usr/share/sddm/scripts/Xstop"`
`[11:19:03.510] (WW) DAEMON: QProcess: Destroyed while process ("/usr/libexec/sddm-helper") is still running.`

**Comment by anselmolsm** [comment]
> My understanding is this sddm bundle was to be built without the Plasma display

You're correct.

> [ 1895.230] (EE) Failed to open authorization file "/var/run/sddm/{85a589ae-514b-4d03-9e4c-778259068d09}": No such file or directory

It seems we're seeing separate issues here. The first one is about error above happened on my bare metal CL setup (running KDE/plasma) once yesterday, I haven't explored further enough but a 
 first guess would be a race condition between sddm service and systemd's services that setup sysuser and tmpfiles. As I said, it needs further verification. And the other issue is the lxqt startup.

**Comment by 0n0w1c** [comment]
You are correct. I just did a fresh install of the desktop iso. I then installed desktop-lxqt and copied over a menu.xml for openbox.

If I use gdm to select LXQt (with openbox), the root window/menu has artifacts. If I select openbox only, the root window/menu remains free of artifacts.

How best to handle this ticket? Close it and open 2 new ones? One for SDDM and one for LXQt?

**Comment by 0n0w1c** [comment]
I do have an older version of desktop-lxqt installed on a VM under VMware Fusion. I do not have the artifacts issue, so this could be caused by a recent change, unsure. Maybe it would be helpful to help to debug?

**Comment by anselmolsm** [comment]
> How best to handle this ticket? Close it and open 2 new ones? One for SDDM and one for LXQt?

I think it makes sense to create a separate one to track LXQt startup and leave this one open to continue tracking the sddm issue(s).


> Maybe it would be helpful to help to debug?

Not sure, but if you could share the CL older and newer versions, that would help to check if any recent change causes the problem. Thanks!
